### DISCUSSION TITLE
Welcome to nHapi Discussions!

### DISCUSSION THREAD

**Discussion by milkshakeuk** [nHapiNET/nHapi#290] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tdcnx** [comment]
Hi,

We are trying to integrate ASTM support into nHapi.
Is this something you would consider to be integrated into nHapi?

**Reply by milkshakeuk** [reply]
Possibly, you would have to tell us more about it.
